### ISSUE TITLE
When the connection is dropped, the video window remains opened for remote participants

### ISSUE THREAD

**Issue by mconf-daileon** [mconf/bigbluebutton#84]
---

Author Name: **Felipe Cecagno** (Felipe Cecagno)
Original Redmine Issue: 707, http://dev.mconf.org/redmine/issues/707
Original Date: 2013-04-03

---

It doesn't display the video anymore, but it keeps opened.

**Comment by mconf-daileon** [comment]
---

Original Redmine Comment
Author Name: **Felipe Cecagno** (Felipe Cecagno)
Original Date: 2013-07-19T12:52:38Z

---

This is very easy to reproduce if you try the system in a low bandwidth environment. Instead of dropping video packets, Red5 just queue them until the queue is full, then it drops the connection, but the user doesn't get any notification about it.

**Comment by mconf-daileon** [comment]
---

Original Redmine Comment
Author Name: **Felipe Cecagno** (Felipe Cecagno)
Original Date: 2013-07-19T12:54:10Z

---

The first step of this implementation should be to notify the client that the connection was dropped and close the window. Another issue will be created to address the second part.

**Comment by mconf-daileon** [comment]
---

Original Redmine Comment
Author Name: **Maurício Cruz** (Maurício Cruz)
Original Date: 2013-08-08T18:42:21Z

---

Could not reproduce this bug on BBB .81.

I tested with a simulated connection using WANem with the fallowing parameters: 1000 ms delay, 40 Kbps and 10% loss.

The results was a less than 1 frames per second video with a huge delay, as expected. When the network limitation was removed, the video returned to normal after some time.
